packages/react-devtools-shared/src/backend/DevToolsOwnerStack.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+// This is a DevTools fork of ReactFiberOwnerStack.
+
+// TODO: Make this configurable?
+const externalRegExp = /\/node\_modules\/|\(\<anonymous\>/;
```

Comment:
These filters will likely just be removed and left to ignore listing in the browser, but this is just in line with what the implementation does elsewhere for now.